roles/kubernetes-apps/metrics_server/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+metrics_server_kubelet_insecure_tls: true
+metrics_server_kubelet_preferred_address_types: "InternalDNS,InternalIP,Hostname,ExternalDNS,ExternalIP"
```

Comment:
Why not do this:

`metrics_server_kubelet_preferred_address_types: "{{ kubelet_preferred_address_types }}"`